frontend/src/metabase/modes/types.ts

```diff
@@ -106,11 +113,25 @@ export const isQuestionChangeClickAction = (
   clickAction: ClickAction,
 ): clickAction is QuestionChangeClickAction => "question" in clickAction;
 
-export const AlwaysDefaultClickAction = (
+export const isAlwaysDefaultClickAction = (
   clickAction: ClickAction,
 ): clickAction is AlwaysDefaultClickAction =>
   "defaultAlways" in clickAction && clickAction.defaultAlways;
 
 export const isRegularClickAction = (
   clickAction: ClickAction,
-): clickAction is RegularClickAction => !AlwaysDefaultClickAction(clickAction);
+): clickAction is RegularClickAction =>
+  !isAlwaysDefaultClickAction(clickAction);
+
+export interface ModeFooterComponentProps {
```

Comment:
Backport this change from https://github.com/metabase/metabase/pull/33143/files#diff-1ceab4c6081d4d5800344c87d9320b7a36cf041576d69ea7b5c614a90b6d2d93R141-R152